CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+Change log
+-----------
+
+# v2.7.4+rev1 
+## 2017-10-19
+
+* Initial release for Siemens IOT2000 [Theodor]
```

Comment:
Wrong commit log

Reply:
git commit log still says v2.7.2+rev1
Please pay attention, previous comment said this already

Reply:
This should say 2.7.6